extensions/tmtv/src/Panels/PanelPetSUV.tsx

```diff
@@ -89,8 +90,8 @@ export default function PanelPetSUV({ servicesManager, commandsManager }) {
 
   // get the patientMetadata from the StudyInstanceUIDs and update the state
   useEffect(() => {
-    const { unsubscribe } = DisplaySetService.subscribe(
-      DisplaySetService.EVENTS.DISPLAY_SETS_ADDED,
+    const { unsubscribe } = cornerstoneCacheService.subscribe(
```

Comment:
And listen for the changes on the event that is actually fired when the values are changed, rather than an unrelated event that may or may not be fired at the right time.  
Note that this fixes a theoretical issue if the display sets being displayed got updated - something we will probably see in the future.